webplugin/scss/components/_km-chat-popup.scss

```
@@ -76,6 +79,11 @@ $popupClass: '.chat-popup-widget-container';
     & .chat-popup-widget-close-btn svg {
         width: 14px;
         height: 14px;
+        display: block;
+    }
+
+    & .chat-popup-widget-close-btn svg path {
+        fill: currentColor;
```

Stylelint (17.13.0)

[error] Expected "currentColor" to be "currentcolor" (value-keyword-case)

(value-keyword-case)